polyfill/lib/duration.mjs

```diff
@@ -53,17 +53,17 @@ export class Duration {
         [years, months, days, hours, minutes, seconds, milliseconds, microseconds, nanoseconds] = arr;
         break;
       case 'balance':
-        let tdays;
+        let deltaDays;
         ({
-          days: tdays,
+          deltaDays,
```

Comment:
Would this work as expected? Do tests pass?

Reply:
Yes, this wasn't covered by any tests before, so it wasn't obvious that it was broken. The new tests that I added do cover it.